### PR TITLE
shrink bundle

### PR DESCRIPTION
**Pull request by turadg**
Refs:
- https://github.com/Agoric/agoric-sdk/issues/9900

**Before**
```
  ✔ [expected fail] bundles small enough for Mainnet
    ℹ {
        bundleName: 'orca',
        compressedSize: '0.552 MB',
        originallySize: '2.356 MB',
      }
```

**After**
```
  ✔ bundles small enough for Mainnet (3.5s)
    ℹ {
        bundleName: 'orca',
        compressedSize: '0.361 MB',
        originallySize: '1.841 MB',
      }
```

### CODE REVIEW

File: contract/test/build-proposal.test.js [hunk 1]
```diff
@@ -20,7 +22,10 @@ test.before(async t => {
    * @param {string} name of an _already cached_ bundle
    */
   const compressBundle = async name => {
-    const bundle = await bundleCache.load('', name);
+    // NOTE load options must match those used in the proposal builder
+    const bundle = await bundleCache.load('', name, trace, {
+      elideComments: true,
+    });
```
Comment (dckc) [inline contract/test/build-proposal.test.js:28]: Nicely done. Thanks for maintaining this prototype tool, even after applying `elideElements` uniformly throughout agoric-sdk.

File: contract/test/build-proposal.test.js [hunk 2]
```diff
@@ -53,8 +58,7 @@ test.before(async t => {
   t.context = { compressBundle, $, runPackageScript, listBundles };
 });
 
-// Will not fit on Mainnet, but that's okay for current purposes
-test.failing('bundles small enough for Mainnet', async t => {
+test('bundles small enough for Mainnet', async t => {
```
Comment (dckc) [inline contract/test/build-proposal.test.js:61]: :clap:

**Review by dckc — APPROVED** [review]
nice work!

File: contract/test/build-proposal.test.js (hunk 1 above)
Comment (erights) [reply contract/test/build-proposal.test.js:28]: @dckc ,
`elideElements`?

Comment (dckc) [reply contract/test/build-proposal.test.js:28]: typo for `elideComments`

**Pull request merged**